Scripts/test-CocoaPods-integration.sh

```diff
@@ -1,37 +1,51 @@
 #!/bin/bash
 
+# Any subsequent(*) commands which fail will cause the shell script to exit immediately
+set -eo pipefail
+
 function print_help {
   echo "Test CocoaPods Integration"
   echo " "
   echo "test-CocoaPods-integration [-w]"
   echo " "
   echo "options:"
   echo "-w, --exclude-wechat      exclude wechat module"
+  echo "-t, --team                set DEVELOPMENT_TEAM to all bundle modules"
 }
 
-set -e # Any subsequent(*) commands which fail will cause the shell script to exit immediately
+function echo_header {
+  echo " "
+  echo "############# $1 #############"
+}
 
-INCLUDE_WECHAT=true
+INCLUDE_WECHAT=false
```

Comment:
why is this changed?, we chat should be included by default

Reply:
the logic was inverted..